amivapi/users/security.py

```diff
@@ -105,6 +105,22 @@ def hide_fields(response):
                     item.pop(key)
 
 
+# Project password status
+
+def project_password_status(response):
```

Comment:
This function can only work if it is applied *before* hide_fields, otherwise there will be no password field. This should be mentioned somewhere.

Reply:
done here and also in __init__.py